spec/factories/location.rb

```diff
@@ -22,7 +22,7 @@
     end
 
     trait :incomplete do
-      units { nil }
+      mobility_type { nil }
       confirmed { false }
     end
```

Comment:
Use a different attribute in the location factory is make it incomplete

Reply:
do we need to? I mean Idm which attribute it is, just curious why it needs changing

Reply:
No we don't I've changed it back, early on a test would fail because of this but not anymore.